.github/workflows/nightly-tests.yml

```diff
@@ -156,16 +156,16 @@ jobs:
           rm -rf results
           mkdir -p results
 
-      - name: Run guest benchmarks
-        run: |
-          source .venv/bin/activate
-          bash ./openvm/scripts/run_guest_benches.sh
+      #- name: Run guest benchmarks
+      #  run: |
+      #    source .venv/bin/activate
+      #    bash ./openvm/scripts/run_guest_benches.sh
 
       - name: Checkout openvm-reth-benchmark
         uses: actions/checkout@v4
         with:
           repository: powdr-labs/openvm-reth-benchmark
-          ref: main
+          ref: 409ce99abce59ce3ee9f0adbfcb56f29627b5952
```

Comment:
This must have been the commit on 07-30, see: https://github.com/powdr-labs/openvm-reth-benchmark/commits/main/